src/module.js

```diff
@@ -1,8 +1,7 @@
 const spawn = require('child_process').spawn;
 const spawnargs = require('spawn-args');
 
-// eslint-disable-next-line padding-line-between-statements
-const execute = (command, log) => {
+function execute (command, log) {
```

Comment:
I wonder if this is maybe a change that was automatically applied by using the --fix option. Did you change that intentionally?

Reply:
Yes, can you please revert the changes.

Reply:
My apologies, I thought your comment was referring to _this_ line:
```
// eslint-disable-next-line padding-line-between-statements
```
I purposely made the change to the `execute` function definition itself because I needed to invoke it from my unit tests. The `rewire` module allows me to invoke non-exported module functions like `execute`, but only if they are written in the `function` syntax, not in the `const` syntax.
I could move `execute` into its own exported module instead?